cdc/processor/pipeline/sorter.go

```diff
@@ -102,6 +114,45 @@ func (n *sorterNode) Init(ctx pipeline.NodeContext) error {
 				if msg == nil {
 					continue
 				}
+				if msg.RawKV != nil && msg.RawKV.OpType != model.OpTypeResolved {
```

Comment:
`if msg.RawKV != nil && msg.RawKV.OpType != model.OpTypeResolved` and `if msg.RawKV != nil && msg.RawKV.OpType == model.OpTypeResolved` is quite verbose, and only one would be true, maybe this logic can be simplified ? how about check `msg.RawKv == nil` just continue, and check `OpType` further ?

Reply:
Yes, sure